src/actions/breakpoints.js

```diff
@@ -60,7 +55,6 @@ export function syncBreakpoint(
 
     return dispatch({
       type: "SYNC_BREAKPOINT",
-      breakpoint,
       [PROMISE]: syncPromise
```

Comment:
I'm glad we waited on this abstraction, i'm beginning to think we just want to move complicated or all actions to their own files

perhaps we should inline the call

```js
dispatch({
  type: "SYNC_BREAKPOINT",
  [PROMISE]: syncClientBreakpoint() 
})
```

or and this is a radical idea... 

move the whole thing to the syncBreakpoint file

```js
export function syncBreakpoint(..) {
  return ({ dispatch, getState, client, sourceMaps }: ThunkArgs) => {
    dispatch( 
      type: "SYNC_BREAKPOINT",
      [PROMISE]: syncClientBreakpoint() 
    )
  })
}
```

The advantage of this approach is that it's intellectually honest. We're just saying, go over there to see this action.


anyways, one big nit